input-output-hk/lace
# feat(staking): add multidelegation beta banner/modal

# Checklist

- [x] JIRA - LW-7633

---

## Proposed solution

This PR introduces simple banner/modal informing users about the Beta release of multidelegation feature.

## Testing

1. Open the staking tab
2. Observe the beta banner/modal
3. Wait 2sec until the "got it" button is enabled and click it
4. Beta banner should not be presented anymore while visiting staking

*In order to test the flow again you need to remove/clear the localStorage key `multidelegationFirstVisit`. You can do it manually or by console command `localStorage.removeItem('multidelegationFirstVisit')`.

## Screenshots

<img width="523" alt="image" src="https://github.com/input-output-hk/lace/assets/17825044/f9bd0015-d675-477c-8dc9-53d22b321bee">
<img width="507" alt="image" src="https://github.com/input-output-hk/lace/assets/17825044/6949e36b-c9a5-4b6e-a8a1-cd4fe6ebba1c">



---
# Allure report
`allure-report-publisher` generated test report!



**smokeTests**: ❌ [test report](https://lace-qa:8PP5wBaDV6UoXj@dq4ajm5i5q7bz.cloudfront.net/linux/chrome/1325/5761359357/index.html) for [fbd005ca](https://github.com/input-output-hk/lace/pull/348/commits/fbd005caf7a68ecb2fa686e151ab526ae7643e0a)
|       | passed | failed | skipped | flaky | total | result |
|-------|--------|--------|---------|-------|-------|--------|
| Total | 35     | 1      | 0       | 0     | 36    | ❌     |
